packages/react-vapor/src/components/tab/tests/TabSelectors.spec.tsx

```diff
@@ -47,5 +47,16 @@ describe('TabSelector', () => {
             const expectedTabGroup = tabGroupState.tabs[0];
             expect(tabGroup).toEqual(expectedTabGroup);
         });
+        it('should return true if a tab is selected, false if not', () => {
```

Comment:
The case where the tab with the specified id doesn't exist in the state is not covered (it should return false).